src/Pecee/SimpleRouter/Router.php

```diff
@@ -371,6 +377,9 @@ public function routeRequest(): ?string
             foreach ($this->processedRoutes as $key => $route) {
 
                 $this->debug('Matching route "%s"', get_class($route));
+                
+                /* Add current processing route to constants */
+                $this->currentProcessingRoute = $route;
```

Comment:
Why are you initializing `$currentProcessingRoute` as an array, but set and use it like a normal single Route object?
referring to:
 `$router->getCurrentProcessingRoute()->getName();` when return type is an array `getCurrentProcessingRoute(): array`

Reply:
Oh yeah. It's a bit confused.
I updated my PR. Thanks!